autogen/main/cluster.tf.tmpl

```diff
@@ -612,6 +612,15 @@ resource "google_container_node_pool" "pools" {
     }
   }
 
+  {% if beta_cluster %}
+  dynamic "placement_policy" {
+    for_each = length(lookup(each.value, "placement_policy", "")) > 0  ? [each.value] : []
+    content {
+      type = lookup(placement_policy.value, "placement_policy", null)
+    }
+  }
+  {% endif %}
```

Comment:
It looks like the block right below this is also a beta conditional, can we merge both?